rbac/management/management/commands/utils.py

```diff
@@ -205,7 +206,16 @@ def batch_import_workspace(records):
                     modified=record["modified_on"],
                 )
                 workspaces.append(workspace)
-            pairs.append((str(workspace.id), str(parent.id)))
+
```

Comment:
According to previous comment - we can skip imports - so we don't need to change it.